pegasus/sites.v3/code.org/views/video_player.js.haml

```diff
@@ -1,13 +1,12 @@
-%script{src: "/js/videojs-ie8.min.js"}
-%script{src: "/js/video.min.js"}
-
 %link{href: "/css/video-js.css", rel: "stylesheet"}
 
 :javascript
 
   $(window).load(function() {
     if (window.location.search.indexOf("force_youtube_fallback") !== -1)
     {
+      $.getScript("/js/videojs-ie8.min.js");
+      $.getScript("/js/video.min.js");
```

Comment:
Thanks for the suggestions. 
1.  Your change makes sense, I am testing that out now
2.   You are correct getScript call is asynchronous - it can be made synchronous. The getScript calls could be changed to $.ajax({
  url: 'js/video.min.js',
  async: false,
  dataType: "script",
});
I will also test out your suggestion with the delaying the script being added to the DOM and will let you know the results